core/server/data/migrations/versions/2.4/1-add-webhook-permission-roles.js

```diff
@@ -0,0 +1,55 @@
+const _ = require('lodash');
```

Comment:
> core/server/data/migrations/versions/2.4/

Why did you choose 2.4? Ghost's package.json version is already 2.5. That means we need to use 2.6, which is the next minor :)

Reply:
🙈 Yes my bad, updated to 2.6!